### PR TITLE
Unit Test for Magento\LayeredNavigation\Observer\Edit\Tab\Front\ProductAttributeFormBuildFrontTabObserver

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Covers Magento\LayeredNavigation\Observer\Edit\Tab\Front\ProductAttributeFormBuildFrontTabObserver class with a unit test.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Please fix static tests.

File: app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+     */
+    public function testExecuteWhenOutputEnabled()
+    {
+        $this->moduleManagerMock->expects($this->once())
+            ->method('isOutputEnabled')
+            ->with('Magento_LayeredNavigation')
+            ->willReturn(true);
+
+        $fieldsetMock = $this->createMock(Fieldset::class);
+        $fieldsetMock->expects(self::exactly(3))->method('addField');
```
Comment (dmytro-ch) [inline app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php]: Could you please replace the static method's call for consistency with other stubs definition.
```suggestion
        $fieldsetMock->expects($this->exactly(3))->method('addField');
```

File: app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                'optionList' => $this->optionListLock,
+                'moduleManager' => $this->moduleManagerMock,
+            ]
+        );
+    }
+
+    /**
+     * Test case when module output is disabled
+     */
+    public function testExecuteWhenOutputDisabled()
```
Comment (dmytro-ch) [inline app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php]: It's not critical but would be preferred to follow the strict type of definition for methods signature.
```suggestion
    public function testExecuteWhenOutputDisabled(): void
```

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hello @karyna-tsymbal-atwix, could you please fix minor CR recommendations?
Thank you!

File: app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php (hunk 1 above)
Comment (karyna-t) [reply app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php]: @dmytro-ch  thanks for the note, fixed!

File: app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php (hunk 2 above)
Comment (karyna-t) [reply app/code/Magento/LayeredNavigation/Test/Unit/Observer/Edit/Tab/Front/ProductAttributeFormBuildFrontTabObserverTest.php]: done

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6912](https://jira.corp.magento.com/browse/ENGCOM-6912) has been created to process this Pull Request

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

The test is added and pass 
ProductAttributeGridObserverTest

**Pull request merged**